lib/init.js

```diff
@@ -166,7 +166,9 @@ module.exports = {
         }
 
         const linter = new (await getPugLint(projectDir))();
+        linter._basePath = projectDir; // eslint-disable-line no-underscore-dangle
```

Comment:
It looks like if you write this as `linter['_basePath'] = projectDir;` you can avoid the rule disabling entirely.